Source/WebCore/bindings/js/JSDOMOperationReturningPromise.h

```diff
@@ -55,6 +55,27 @@ class IDLOperationReturningPromise {
         }));
     }
 
+    using Operation2 = JSC::EncodedJSValue(JSC::JSGlobalObject*, JSC::CallFrame*, ClassParameter, Ref<DeferredPromise>&&, Ref<DeferredPromise>&&);
+    template<Operation2 operation, CastedThisErrorBehavior shouldThrow = CastedThisErrorBehavior::RejectPromise>
+    static JSC::EncodedJSValue callPair(JSC::JSGlobalObject& lexicalGlobalObject, JSC::CallFrame& callFrame, const char* operationName)
```

Comment:
This could be given a better name, like "callReturningPromisePair"

Reply:
You are right, I failed to take a clue from the method names below it. Fixed.